flagparse/flags.go

```diff
@@ -41,18 +41,33 @@ type UploadConfig struct {
 	client.UploadableConfig
 	logger.LogConfig
 
-	Files string `json:"files,omitempty" descr:"Glob pattern for the files to upload"`
+	Files string            `json:"files,omitempty" descr:"Glob pattern for the files to upload"`
+	Mode  client.AccessMode `json:"mode,omitempty" def:"strict" descr:"{mode}"`
 }
 
 //ConfigNames contains template names to be replaced in config properties descriptions and default values
 var ConfigNames = map[string]string{
 	"name": "AutoUploadable", "feature": "Uploadable", "period": "Upload period",
 	"action": "upload", "actions": "uploads", "running_actions": "uploads",
+	"mode": "File access mode. Restricts which files can be requested dynamically for upload through 'upload.files' " +
```

Comment:
We need a ticket to update the file upload reference doc. Otherwise, this PR has to be updated.

Reply:
https://github.com/eclipse-kanto/kanto/issues/61